src/App.tsx

```diff
@@ -14,6 +17,21 @@ function App() {
         onSubmit={() => console.log('submitted')}
       />
       <LongButton onClick={() => null}>Example</LongButton>
+      <ActionButton
+        onPress={() => console.log('clicked ActionButton')}
+        icon={<FaHeart id="kid" />}
```

Comment:
This was a wise solution, but let's reflect about it for a while:

We now have to pass a `id="kid"` every time we wish to use this component. There are a few problems with this approach. It is repetitive and not really documented (a new person would have to go to our codebase and see this component being used in order to know they should use an id, they would have to remember it and it isn't even that clear why we need it, unless they check out the component's styles).

A better solution could be to create a component which automatically creates an icon with the `id="kid"` to the icon. But, again, how to dynamically pass this icon prop? I will give you a suggestion bellow.